ProjFS.Mac/PrjFSKext/PrjFSKext.xcodeproj/project.pbxproj

```diff
@@ -196,6 +196,7 @@
 				TargetAttributes = {
 					C6C780AF207FC67200E7E054 = {
 						CreatedOnToolsVersion = 9.3;
+						ProvisioningStyle = Automatic;
```

Comment:
@sanoursa @nickgra any ideas what this setting does?  At some point xcode decided to add it to the project.

Reply:
I was going to ask you the same thing :-)

Reply:
Did you update your XCode version lately? Apple is reworking some things with signing and provisioning profiles, so this is likely related. We should just take the change now (like we do when VS insists on changing a project metadata file), no point in fighting with the IDE.